targets/targets.json

```diff
@@ -5646,7 +5646,18 @@
             "1101"
         ],
         "program_cycle_s": 6,
-        "bootloader_supported": true
+        "bootloader_supported": true,
+        "supported_application_profiles" : ["full", "bare-metal"],
+        "supported_c_libs": {
+            "arm": [
+                "std",
+                "small"
+            ],
+            "gcc_arm": [
+                "std",
+                "small"
+            ]
+        }
```

Comment:
For all the `targets.json` PRs, please ensure `supported_toolchains` matches `supported_c_libs` for the targets being changed (i.e. no IAR)